presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/Environment.java

```diff
@@ -150,6 +157,11 @@ private Environment tryStart()
 
     public void stop()
     {
+        if (attached) {
```

Comment:
does it relly solve `Could not attach tests to unhealthy environment singlenode-privacera`???

nothing was stopped at my machine, containers were still working as expected, the problem was somewhere in `allCOntainersHealthy` probably

Reply:
I removed that in https://github.com/prestosql/presto/pull/5415/commits/eed10acea50d5e566c00461aefc5c2082fe2ca7c